src/middlewared/middlewared/plugins/kerberos.py

```diff
@@ -166,6 +172,24 @@ async def do_update(self, data):
         await super().do_update(data)
 
         await self.middleware.call('etc.generate', 'kerberos')
+
+        # gssproxy provides support for large kerberos tickets.
+        # It does not replace the kernel gssapi. The kernel will use gssproxy
+        # if it is available.  This is determined during the boot process.
+        # Currently, the status of gssproxy cannot be changed outside of boot time.
+        if new['enable_gssproxy'] != old['enable_gssproxy']:
+            # gssproxy seems to ignore the 'disable' setting and requires masking
+            if new['enable_gssproxy'] is True:
+                await systemd_unit("gssproxy", "unmask")
```

Comment:
There are various places all over our codebase where systemd service enable/disable is handled:
* etc_files/systemd (middleware)
* config upload (middleware)
* installer code (scale-build)

Reply:
removed enable/disable capability